indra/newview/llimview.cpp

```diff
@@ -2727,7 +2727,7 @@ LLIncomingCallDialog::LLIncomingCallDialog(const LLSD& payload) :
 
 void LLIncomingCallDialog::onLifetimeExpired()
```

Comment:
This function looks quite bad in the webview. I suspect it has spaces and tabs mixed. Settling on one would be preferred

Reply:
Should be better now

Reply:
Yes, looks awesome now